demo

```diff
@@ -1,12 +1,30 @@
 #!/bin/bash
 
-
 CLIENT="-f docker-compose.client.yml"
 if [ -n "${NO_CLIENT}" ]; then
     unset CLIENT
     echo "-- disabling client containers"
 fi
 
+if [ "$#" -eq 0 ]; then
+    echo "Arguments passed to this command are passed directly to docker-compose, if you want to run the demo, run:"
+    echo ""
+    echo "'./demo up'"
+    exit 1
+fi
+
+# speed up first start by pulling containers in parallel
+if [[ "$(docker images | grep -q 'mendersoftware/deployments')" -eq 0 ]]; then
+    docker_compose_output=$(docker-compose pull -h)
+    check=$( echo "$docker_compose_output" | grep -q -- '--parallel')
+
+    if [[ "$check" -eq 0 ]]; then
```

Comment:
Same here, you need to call the command in the `if` without the `$()` and the `[[ ]]`.